<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tree</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(229, 227, 221);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(202, 195, 212);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>60</width>
        <height>661</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: rgb(229,227,221);
	line-height:10px;
	font: 22pt &quot;Cascadia Mono&quot;;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_5">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>FastTree</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>500</height>
                 </size>
                </property>
                <property name="text">
                 <string>Models</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="mods">
                <property name="styleSheet">
                 <string notr="true">QComboBox {
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    background: transparent; 
    border: 1px solid gray; 
	border-color: #E5E5E5;
	background-color: #FFFFFF;
}
</string>
                </property>
                <item>
                 <property name="text">
                  <string>GTR</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>JTT</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>LG</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>WAG</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="fasttree">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
background-color: rgb(188, 149, 117);
border:none;
border-radius:7px;
color:white}
QPushButton::hover {
    background-color: rgb(226, 179, 141);
}

QPushButton::pressed {
    background-color: rgb(158, 125, 99);
}</string>
              </property>
              <property name="text">
               <string>Generate</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_4">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>ModelFinder</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="modelfinder_button">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
background-color: rgb(188, 149, 117);
border:none;
border-radius:7px;
color:white}
QPushButton::hover {
    background-color: rgb(226, 179, 141);
}

QPushButton::pressed {
    background-color: rgb(158, 125, 99);
}</string>
              </property>
              <property name="text">
               <string>Generate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="model_results">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
background-color: rgb(188, 149, 117);
border:none;
border-radius:7px;
color:white}
QPushButton::hover {
    background-color: rgb(226, 179, 141);
}

QPushButton::pressed {
    background-color: rgb(158, 125, 99);
}</string>
              </property>
              <property name="text">
               <string>Results</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pic">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: rgb(188, 149, 117);
border:none;
border-radius:7px;
color:white}
QPushButton::hover {
    background-color: rgb(226, 179, 141);
}

QPushButton::pressed {
    background-color: rgb(158, 125, 99);
}</string>
        </property>
        <property name="text">
         <string>View Pictures</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
